public/app/features/explore/TraceView/components/utils/filter-spans.tsx

```diff
@@ -133,10 +131,11 @@ const getTagMatches = (spans: TraceSpan[], tags: Tag[]) => {
             (span.traceState && tag.key === TRACE_STATE) ||
             tag.key === ID
           ) {
-            return getReturnValue(tag.operator, true);
+            return tag.operator === '=' || tag.operator === '=~' ? true : false;
```

Comment:
O wow, this is quite neat hidden feature to include or exclude spans with certain tags (when no value is provided) just by changing the operator (and keeping the value empty). Wonder if users are aware of it :D

Reply:
100% this is actually quite a handy way to search.I think it's one of those features that users are pleasantly surprised by when they first select a tag and notice it updates the selected spans in the UI, before they've even selected a value.